src/main/resources/org/cbioportal/legacy/persistence/mybatis/StudyMapper.xml

```diff
@@ -142,23 +143,25 @@
         </include>
         <if test="projection == 'SUMMARY' || projection == 'DETAILED'">
             ,
-            COUNT(CASE WHEN sample_list.STABLE_ID = CONCAT(cancer_study.CANCER_STUDY_IDENTIFIER,'_all') THEN 1 ELSE NULL END) AS allSampleCount
+            <!--TODO : FIX-->
+            <!--COUNT(CASE WHEN sample_list.stable_id = CONCAT(cancer_study.cancer_study_identifier,'_all') THEN 1 ELSE NULL END) AS allSampleCount-->
+            1 AS allSampleCount
```

Comment:
I think this is fixed in the new clickhouse implementation, so we don't use this `getStudies` mapping anymore, right?